README.md

```diff
@@ -115,6 +115,8 @@ _cscope_maps_ comes with following defaults:
     picker_opts = {
       window_size = 5, -- any positive integer
       window_pos = "bottom", -- "bottom", "right", "left" or "top"
+      -- options for Snacks picker (---@class snacks.picker.Config)
+      snacks_opts = {}, -- pass-through options for of Snacks picker -- passed to Snacks.picker()
```

Comment:
would it be better if  `snacks_opts` is renamed to `snacks` considering it's already inside `picker_opts`?
```suggestion
      -- pass-through options for of Snacks picker
      snacks = {}, -- snacks config
```